packages/release/bottlerocket.mount.in

```
@@ -0,0 +1,19 @@
+[Unit]
+Description=Mount Bottlerocket persistent data directory (/.bottlerocket)
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+After=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dPRIVATE.device
+Requires=dev-disk-by\x2dpartlabel-BOTTLEROCKET\x2dPRIVATE.device
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Mount]
+What=/dev/disk/by-partlabel/BOTTLEROCKET-PRIVATE
+Where=/.bottlerocket
+# "noexec" omitted to allow netdog to run
+Options=defaults,nosuid,nodev,noatime,private,context=system_u:object_r:private_t:s0
```

Any time these are both true:
1. You are applying a `context=` mount option.
2. You are in a unit with `DefaultDependencies=no`.

Then you must add these dependencies:
```
After=selinux-policy-files.service
Wants=selinux-policy-files.service
```

Otherwise the context option can be silently dropped, if `mount` doesn't see the policy files and concludes that SELinux is not actually enabled.